packages/combo-box/src/vaadin-combo-box-mixin.d.ts

```diff
@@ -177,6 +184,4 @@ export declare class ComboBoxMixinClass<TItem> {
   checkValidity(): boolean;
 
   protected _revertInputValue(): void;
-
-  protected _getItemElements(): HTMLElement[];
```

Comment:
This method is now private because the extensions don't need to override it thanks to `_tagNamePrefix` usage.